public/js/clients/chrome.js

```diff
@@ -39,10 +41,15 @@ function createTabs(tabs) {
 
 function connectClient() {
   const deferred = defer();
-  fetch("/chrome-tabs").then(res => {
-    res.json().then((body) => {
-      deferred.resolve(createTabs(body));
-    });
+
+  if(!isEnabled("chrome.debug")) {
+    return deferred.resolve(createTabs([]))
+  }
```

Comment:
Why do this here? We don't have a corresponding `firefox.debug` check in the firefox client, so it seems better to have it on the server.